contributing/code/bc.rst

```diff
@@ -310,6 +314,10 @@ Change return type                                  Yes             Yes
 **Static Methods**
 Turn non static into static                         No              No
 Turn static into non static                         No              No
+**Constants**
+Add constant                                        Yes             Yes
+Remove constant                                     No              No
+Change value of a constant                          Yes             Yes
```

Comment:
Well, having constants is handy when using subscribers (I always do rather than registering listeners in the service config, and Symfony does too). It makes it less likely to make typos in event names (a typo in the constant name would be a PHP error)